doc/usermanual/darkroom/concepts/colormanagement.xml

```diff
@@ -24,15 +24,16 @@
       <listitem><para>
         darktable automatically reads the display profile of your monitor (if properly
         configured) for an accurate color rendition on screen. Multi-screen setups are fully
-        supported as long as a system service like <emphasis>colord</emphasis> is in place and
-        properly set up to inform darktable about the correct monitor profile.
+        supported as long as a system service like
+        <ulink url="https://www.freedesktop.org/software/colord/intro.html"><emphasis>colord</emphasis></ulink>
+        is in place and properly set up to inform darktable about the correct monitor profile.
       </para></listitem>
 
       <listitem><para>
         Output files can be encoded in one of darktable's built-in profiles, like
```

Comment:
In output profile I would also list: ProPhoto, Rec2020, Rec709. Maybe not PQ P3 RGB, HLG P3 RGB, which are not main stream.

Reply:
Done. Also another wikipedia link and minor copy edits.